manifests/server/config_entry.pp

```diff
@@ -130,10 +136,13 @@
     } elsif $name == 'data_directory' {
       # We need to force postgresql to stop before updating the data directory
       # otherwise init script breaks
+      $stop_command = ['service', shell_escape($postgresql::server::service_name), 'stop']
+      $stop_onlyif = ['service', shell_escape($postgresql::server::service_name), 'status']
+      $stop_unless = ["grep 'PGDATA=${shell_escape($value)}' /etc/sysconfig/pgsql/postgresql"]
```

Comment:
Shouldn't this also be an array with multiple values?
```suggestion
      $stop_unless = ['grep', "PGDATA=${shell_escape($value)}", '/etc/sysconfig/pgsql/postgresql']
```

Reply:
As above